articles/postgresql/howto-hyperscale-create-users.md

```diff
@@ -13,10 +13,8 @@ ms.date: 1/8/2019
 
 > [!NOTE]
 > The term "users" refers to users within a Hyperscale (Citus)
-> server group. To learn instead about Azure subscription users and their
-> privileges, visit the [Azure role-based access control (Azure RBAC)
-> article](../role-based-access-control/built-in-roles.md) or review [how to
-> customize roles](../role-based-access-control/custom-roles.md).
+> server group. Hyperscale Citus doesn't support role based access control , to know more see [RBAC in Hyperscale Postgresql](https://docs.microsoft.com/en-us/azure/postgresql/postgresql-hyperscale-security-baseline#31-maintain-an-inventory-of-administrative-accounts)
```

Comment:
Two terminology changes:

* Hyperscale Citus → Hyperscale (Citus)
* Hyperscale Postgresql → Azure Database for PostgreSQL - Hyperscale (Citus)

And two miscellaneous changes:

* Remove space before comma
* Remove `/en-us` from docs link (so it can detect the visitor's language

Reply:
@aravindmahadevan-ms please take a look to make adjustments based on Joe's feedback. There is an SLA so please act fast if you plan to keep using the public repo.

Reply:
I made these revisions (and added an introductory sentence, per the CG PR quality criteria).

Reply:
Thanks, it looks good to me now.